src/Parsers/Access/ASTGrantQuery.h

```diff
@@ -26,6 +26,9 @@ class ASTGrantQuery : public IAST, public ASTQueryWithOnCluster
     bool admin_option = false;
     bool replace_access = false;
     bool replace_granted_roles = false;
+    bool current_grants = false;
+    bool with_grant_option = false;
```

Comment:
Please rename it to `grant_option` and use it in `ASTGrantQuery::formatImpl()` (currently `ASTGrantQuery::formatImpl()` can use both `grant_option` and `with_grant_option` causing unnecessary confusion).

Reply:
I ended up completely removing this field.